### ISSUE TITLE
Sections with repeating digits.

### ISSUE THREAD

**Issue by arknave** [Z1MM32M4N/latex-homework-class#1]
Hey, big fan of the blog. I use this class for my homeworks and the section numbering just displayed a 0 if the section was two repeating digits (11, 22, 66, etc.) Here's a document that reproduces the issue for me. Using TexLive 2015 on OSX.

Anyone else having this issue? \writtensection 10 and 12 work fine, but 11 renders as 

`0 Question 1`

Instead of 

`Question 11.1`

as I'd expect. 

```latex
\begin{document}

\maketitle

\renewcommand{\writtensection}{11}

\setcounter{questionCounter}{0}

\hypertarget{section}{%
\section{}\label{section}}

Question content here

\end{document}
```

**Comment by arknave** [comment]
Found a fix, submitted PR.

**Comment by jez** [comment]
Closed via #2.
